legend-engine-xt-analytics-store-entitlement/pom.xml

```diff
@@ -22,14 +22,11 @@
         <version>4.6.3-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
-    <name>Legend Engine - XT - Analytics - Store - Entitlement</name>
     <artifactId>legend-engine-xt-analytics-store-entitlement</artifactId>
```

Comment:
please check with other POMs, I think we need `<packaging>jar</packaging>` as well